Add Antora docs with the Decidim documentation format (#99)
decidim/decidim-bulletin-board: docs/modules/develop/pages/c4_component.adoc

[plantuml]
....
@startuml
!includeurl https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/v2.0.1/C4_Component.puml

' uncomment the following line to make proposals
'LAYOUT_AS_SKETCH()

title System Component diagram for Decidim Bulletin Board (https://decidim.org)

Container_Boundary(ruby_client, "Ruby client") {
    Component(graphql_ruby_client, "GraphQL client", "Ruby", "Allows authority to connect to the BB server.")

>     Component(common, "Common code", "Ruby", "Shared code between Decidim and the BB server: criptography settings, JWT, message identifiers format and parse, etc.")

Reviewer: ```suggestion
    Component(common, "Common code", "Ruby", "Shared code between Decidim and the BB server: cryptography settings, JWT, message identifiers format and parse, etc.")
```

    Component(main, "JS main", "JS", "Implements trustees and voters logic for all voting schemes.")
    Component(graphql_js_client, "GraphQL client", "JS", "Allows trustees and voters to connect to the BB server.")
    Component(dummy_wrappers, "Dummy Wrappers", "JS", "Trustees and voters wrappers for the Dummy voting scheme.")
    Component(eg_wrappers, "ElectionGuard Wrappers", "JS", "Trustees and voters wrappers for the ElectionGuard voting scheme.")
}

Container_Boundary(web_api, "GraphQL API") {
    Component(ruby_client_inner, "Ruby client", "Ruby On Rails Engine", "Common code with the Decidim part.")
    Component(graphql_server, "GraphQL server", "Ruby On Rails", "Implements the GraphQL server logic.")
    Component(dummy_wrapper, "Dummy Wrapper", "Ruby", "Bulletin Board wrapper for the Dummy voting scheme.")
    Component(eg_wrapper, "ElectionGuard Wrapper", "Ruby/Python", "Bulletin Board wrapper for the ElectionGuard voting scheme.")
}
ContainerDb(rel_db, "Relational Database", "PostgreSQL", "Stores clients, elections and log entries")
Container(tasks, "Command line tasks", "Ruby on Rails 6", "Allows to register Decidim instances as authorities.")

> Container(worker, "Worker", "Ruby on Rails 6", "ActiveJob queues for non syncronuos jobs. Works for processing most of the received messages.")

Reviewer: ```suggestion
Container(worker, "Worker", "Ruby on Rails 6", "ActiveJob queues for asynchronous jobs. Works for processing most of the received messages.")
```

> Container(verifier, "Command line application", "Python", "Verify the complete messages log of an election.")

Reviewer: ```suggestion
Container(verifier, "Command line application", "Python", "Verifies the complete messages log of an election.")
```
